src/Engine/EmitMSIL/Replication.cs

```diff
@@ -37,6 +36,7 @@ public static IList ReplicationLogic(MethodBase mi, IList args, string[][] repli
             //Turn the replication guides into a guide -> List args data structure
             var partialInstructions = Replicator.BuildPartialReplicationInstructions(partialReplicationGuides);
 
+            var mi = mInfos.FirstOrDefault();
```

Comment:
whats this about? Is this the followup you referred to?

Reply:
Yes.